src/OpenTK/OpenTK.csproj

```diff
@@ -82,6 +82,18 @@
     <AssemblyOriginatorKeyFile>..\..\OpenTK.snk</AssemblyOriginatorKeyFile>
   </PropertyGroup>
   <ItemGroup>
+    <Reference Include="Mono.Cecil, Version=0.9.6.0, Culture=neutral, PublicKeyToken=0738eb9f132ed756, processorArchitecture=MSIL">
```

Comment:
We've modified the CSProj here for some reason, can you figure out why this is needed, or revert this?

From what I can tell, we shouldn't need a runtime dependency on Mono.Cecil, as that's only used by the IL Rewriter.